common/djangoapps/student/tests/test_auto_auth.py

```diff
@@ -61,6 +62,28 @@ def test_create_multiple_users(self):
         self._auto_auth()
         self.assertEqual(User.objects.all().count(), 2)
 
+    def test_invalidate_cache_user_profile_country_updated(self):
```

Comment:
This seems like an odd place to put this test case. Isn't this test for testing the auto auth functionality used in tests? I think the test should live with embargo tests.